searx/static/themes/simple/.eslintrc.json

```diff
@@ -1,28 +0,0 @@
-{
-    "env": {
-        "browser": true,
-        "es2021": true,
-        "node": true
-    },
-    "extends": "eslint:recommended",
```

Comment:
["eslint:recommended" and "eslint:all" no longer accepted in flat config](https://eslint.org/docs/latest/use/migrate-to-9.0.0#-eslintrecommended-and-eslintall-no-longer-accepted-in-flat-config) / IDK how to load/import the `eslint:recommended` configuration in a config file that is loaded by grunt .. @Bnyro @dalf I assume you are more familiar with the JS eco system than mine .. do you have a clue?